tools/gen_usb_descriptor.py

```diff
@@ -256,21 +270,11 @@ def strings_in_order(cls):
     bFunctionSubClass=cdc.CDC_SUBCLASS_ACM,  # Abstract control model
     bFunctionProtocol=cdc.CDC_PROTOCOL_NONE)
 
-# audio_iad = standard.InterfaceAssociationDescriptor(
-#     description="Audio IAD",
-#     bFirstInterface=audio_control_interface.bInterfaceNumber,
-#     bInterfaceCount=len(audio_interfaces),
-#     bFunctionClass=audio.AUDIO_CLASS_DEVICE,
-#     bFunctionSubClass=audio.AUDIO_SUBCLASS_UNKNOWN,
-#     bFunctionProtocol=audio.AUDIO_PROTOCOL_V1)
-
-
 descriptor_list = []
 descriptor_list.append(cdc_iad)
-# descriptor_list.append(audio_iad)
 descriptor_list.extend(cdc_interfaces)
 descriptor_list.extend(msc_interfaces)
-# descriptor_list.append(audio_control_interface)
+descriptor_list.append(audio_control_interface)
```

Comment:
Not sure why this isn't `audio_interfaces`, not just `audio_control_interface`.

Reply:
Comment added. The control interface manages ordering between the audio and midi subinterfaces.